EC-84: Revise Cache, Catalog and Checkout sections (#3)
ezsystems/commerce-documentation: docs/guide/catalog/catalog_configuration.md

The navigation settings are used to build the navigation tree.

> For more details see [Navigation](../navigation/navigation.md).

Reviewer: ```suggestion
For more details, see [Navigation](../navigation/navigation.md).
```

In addition, the parameters for search have to be defined depending on the data provider (see `econtent_search.yml` or [eContent configuration](../econtent/econtent_configuration.md)).

## Configuring the CatalogFactory

> The following example shows the configuration used for the `CatalogFactory` to determine which method creates the `catalogElements` depending on the Content Type

Reviewer: ```suggestion
The following example shows the `CatalogFactory` configuration that determines which method creates the `catalogElements` depending on the Content Type:
```

silver_eshop.default.catalog_factory.ses_product_type: createProductTypeNode
```

> - `ses_category` up to `ses_product_type` are the identifiers used in the corresponding ContentTypes

Reviewer: ```suggestion
- `ses_category` up to `ses_product_type` are the identifiers used in the corresponding Content Types
```
